core/src/main/java/nl/nn/adapterframework/configuration/Configuration.java

```diff
@@ -413,7 +421,7 @@ public void removeStopAdapterThread(Runnable runnable) {
 	}
 
 	/**
-	 * Register an adapter with the configuration.
+	 * Processor that handles a specific kind of messages in a specific way. Comparable to a function in programming languages.
```

Comment:
Dit is er niet duidelijker van geworden. Wat was hier fout aan de originele tekst?

Reply:
De goede uitleg staat in Adapter.java. Ik zeg maar gewoon dat de adapter de adapter is om het kort te houden.